jml/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:centos7
+
+RUN yum -y update
+RUN yum -y install \
+  awscli \
+  git \
+  java-1.8.0-openjdk-devel \
+  libgomp
+
+RUN mkdir -p /usr/local/openjml{,-build}
+WORKDIR /usr/local/openjml-build
+
+# OpenJML
+RUN git clone https://github.com/OpenJML/OpenJML.git
+RUN cd OpenJML && git checkout e4dc6371e2cf45d1a08b5c91990e2a875f33dd27
+RUN cp OpenJML/OpenJML/releaseBuilds-CI/openjml.jar /usr/local/openjml/openjml.jar
+
+# Solvers
+RUN git clone https://github.com/OpenJML/Solvers.git
+RUN cd Solvers && git checkout 023d9682b6fa2e5d70e7587e2e293bdeae4e0885
+RUN cp -r Solvers/Solvers-linux /usr/local/openjml
+
+WORKDIR /usr/local/openjml
+ENTRYPOINT ["java", "-Xmx12g", "-jar", "openjml.jar", "--", "-Werror", "-progress", "-timeout", "600", "-esc", "-spec-math=bigint", "-code-math=safe", "-minQuant", "-no-staticInitWarning", "-prover", "z3_4_3"]
```

Comment:
Does travis provide 12GB of RAM in its containers?

Reply:
Nice catch.  `trusty` gives 4g.  https://docs.travis-ci.com/user/reference/overview/